sonic-xcvrd/xcvrd/cmis/cmis_manager_task.py

```diff
@@ -893,10 +893,13 @@ def task_worker(self):
                                self.port_dict[lport]['forced_tx_disabled'] = True
                                txoff_duration = self.get_cmis_dp_tx_turnoff_duration_secs(api)
                                self.log_notice("{}: Tx turn off duration {} secs".format(lport, txoff_duration))
-                               self.update_cmis_state_expiration_time(lport, txoff_duration)
                                self.post_port_active_apsel_to_db(api, lport, host_lanes_mask, reset_apsel=True)
                            self.update_port_transceiver_status_table_sw_cmis_state(lport, CMIS_STATE_READY)
                            continue
+
+                        # Arm timer for CMIS_STATE_DP_PRE_INIT_CHECK state
+                        if self.port_dict[lport]['forced_tx_disabled']:
```

Comment:
@prgeor Should we update expiration time only if `self.port_dict[lport]['cmis_expired'] is None`?